### PR TITLE
test(suite-desktop): add tor splascreen test

### PR DESCRIPTION
**Pull request by mroz22**
In this PR I am trying to create a new test for TOR loading #5817 

So far I can tell that:
- it should be doable
- it is not finished yet, tests are still flaky and there are multiple code smells
- it could be that some of the flakiness is in fact caused not by tests but by the code under tests instead.

Suspects to be checked:
- is tor switch in settings turned off  correctly after disabling tor in the splashscreen?
- does tor loading screen always appear? happened to me few times that it did not after enabling tor for the first time and reloading app

### CODE REVIEW

File: packages/components/src/components/loaders/LoadingContent/index.tsx [hunk 1]
```diff
@@ -31,10 +34,18 @@ export const LoadingContent: React.FC<LoadingContentProps> = ({
     children,
     isLoading = false,
     size = 20,
+    dataTestBase,
```
Comment (dahaca) [inline packages/components/src/components/loaders/LoadingContent/index.tsx]: I think it's better to always use the `data-test` name.

File: packages/suite-desktop/src/support/screens/TorLoadingScreen.tsx [hunk 2]
```diff
@@ -177,7 +177,7 @@ export const TorLoadingScreen = ({ callback }: TorLoadingScreenProps) => {
 
     return (
         <ThemeProvider>
-            <Wrapper>
+            <Wrapper data-test="@tor-loading-screen">
```
Comment (dahaca) [inline packages/suite-desktop/src/support/screens/TorLoadingScreen.tsx:172]: I slightly changed the component structure in https://github.com/trezor/trezor-suite/pull/6084, so perhaps you'd have to rebase. The changes will be minimal though ;)

Comment (karliatto) [reply packages/suite-desktop/src/support/screens/TorLoadingScreen.tsx:172]: Thanks for mentioning it, I updated it to the new code.

**Comment by karliatto** [comment]
Closing this because it has been refactored and reopen https://github.com/trezor/trezor-suite/pull/6194.

**Pull request closed without merging**